projects/1.18/assets/dracovita-farm-life/farmlife/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+  "itemGroup.farmlife": "Farm Life",
+
+  "item.farmlife.tribull_shank": "生三足牛排",
+  "item.farmlife.cooked_tribull_shank": "熟三足牛排",
+  "item.farmlife.music_disc_life_on_the_farm": "音乐唱片",
+  "item.farmlife.music_disc_life_on_the_farm.desc": "Mr. Esuoh - Life on the Farm",
+  "item.farmlife.galliraptor_egg": "加利鸡蛋",
+  "item.farmlife.galliraptor": "生加利鸡肉",
+  "item.farmlife.cooked_galliraptor": "熟加利鸡肉",
+  "item.farmlife.tribull_milk": "三足牛奶桶",
+  "item.farmlife.tribull_cheese_wedge": "三足牛奶酪块",
+
+  "item.farmlife.domestic_tribull_spawn_egg": "驯化三足牛刷怪蛋",
+  "item.farmlife.galliraptor_spawn_egg": "加利鸡刷怪蛋",
+
+  "block.farmlife.peacock_burst_poppy": "孔雀蓝色爆裂虞美人",
+  "block.farmlife.electric_burst_poppy": "电光蓝色爆裂虞美人",
+  "block.farmlife.fancy_burst_poppy": "幻想粉色爆裂虞美人",
+  "block.farmlife.olive_burst_poppy": "橄榄绿色爆裂虞美人",
+  "block.farmlife.rusty_burst_poppy": "锈蚀棕色爆裂虞美人",
+  "block.farmlife.sunstreak_burst_poppy": "霞光橙色爆裂虞美人",
+  "block.farmlife.potted_peacock_burst_poppy": "孔雀蓝色爆裂虞美人盆栽",
+  "block.farmlife.potted_electric_burst_poppy": "电光蓝色爆裂虞美人盆栽",
+  "block.farmlife.potted_fancy_burst_poppy": "幻想粉色爆裂虞美人盆栽",
+  "block.farmlife.potted_olive_burst_poppy": "橄榄绿色爆裂虞美人盆栽",
+  "block.farmlife.potted_rusty_burst_poppy": "锈蚀棕色爆裂虞美人盆栽",
+  "block.farmlife.potted_sunstreak_burst_poppy": "霞光橙色爆裂虞美人盆栽",
+  "block.farmlife.tribull_milk_cauldron": "三足牛奶锅",

```

参照原版翻译，`xxx Cauldron` -> `装有xxx的炼药锅`
```suggestion
  "block.farmlife.tribull_milk_cauldron": "装有三足牛奶的炼药锅",
```